### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/layout_moyo-header/
https://panianchuk.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  font-family: Roboto, sans-serif;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:7]: do not apply any styles to the `*` selector except `box-sizing`

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,60 @@
...
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" id="home">
+      <div class="container">
```
Comment (SanyaBratashchuk) [inline src/index.html:16]: why do you need this container?

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,60 @@
...
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <div class="flex">
+      <ul>
+        <li class="item"></li>
+      </ul>
+    </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:64]: Why do you need this?

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Fix your structure, it should look like `<header><nav><ul><li><a> </a> </li>....</ul></nav></header>`

**Comment by panianchuk** [comment]
changed

File: src/style.css [hunk 4]
```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
```
Comment (ivanchuk-yurii) [inline src/style.css:8]: `body` doesn't have default padding

File: src/style.css [hunk 5]
```diff
@@ -1 +1,74 @@
...
+}
+  
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  height: 60px;
```
Comment (ivanchuk-yurii) [inline src/style.css:16]: don't hardcode height

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
add demo and test report links

**Comment by panianchuk** [comment]
https://panianchuk.github.io/layout_moyo-header/
https://panianchuk.github.io/layout_moyo-header/report/html_report/

**Review by alexandra-protyanova — APPROVED** [review]

**Pull request closed without merging**